src/user/room.ts

```diff
@@ -423,7 +460,9 @@ export class Room extends Accessory implements Sayable {
    * @return  {this}                          - this for chain
    *
    * @example <caption>Event:join </caption>
-   * const room = await Room.find({topic: 'event-room'}) // change `event-room` to any room topic in your wechat
+   * const bot = new Wechaty()
+   * await bot.start()
+   * const room = await bot.Room.find({topic: 'event-room'}) // change `event-room` to any room topic in your wechat
```

Comment:
`event-room` is not a good example topic because we have many room events like `room-join`.

Suggest to use `topic of your room` instead